### ISSUE TITLE
How to add more regularization methods?

### ISSUE THREAD

**Issue by qiang-zhang-neu** [MrGiovanni/UNetPlusPlus#32]
This project really help me a lot. 
However, I meet a problem when training our own data that the model is overfitted. Thus, I want to add more regularization methods, like L1/L2 or dropout. How can I add those features?
